src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -35,6 +35,7 @@
     <Compile Include="System\ActivatorTests.cs" />
     <Compile Include="System\ActivatorTests.Generic.cs" />
     <Compile Include="System\AmbiguousImplementationExceptionTests.cs" />
+    <Compile Include="System\AppContext\AppContext.cs" />
```

Comment:
Could you please make this `System\AppContextTests.cs` to follow the pattern established by the existing tests?

Reply:
Ok, I have not noticed those. In that case, this line in .csproj should be moved to the block of existing AppContext tests:
```
    <Compile Include="System\AppContext\AppContext.cs" />
    <Compile Include="System\AppContext\AppContext.Switch.cs" />
    <Compile Include="System\AppContext\AppContext.Switch.Validation.cs" />
```

If you would like to cleanup the file names of the AppContext tests to make them follow the same pattern as most other tests in this project, it would be fine with me too. For example:
```
    <Compile Include="System\AppContextTests.cs" />
    <Compile Include="System\AppContextTests.Switch.cs" />
    <Compile Include="System\AppContextTests.Switch.Validation.cs" />
```